fast/1.0-basic/Tiltfile

```diff
@@ -0,0 +1,13 @@
+docker_build("glitch", ".", dockerfile="glitch.dockerfile", only=['glitch', 'render/api', 'go.mod'],)
+docker_build("storage", ".", dockerfile="storage.dockerfile", only=['storage', 'go.mod'])
+docker_build("muxer", ".", dockerfile="muxer.dockerfile", only=['muxer', 'storage/client', 'storage/api', 'render/api', 'go.mod'])
+
+k8s_yaml([
+    'glitch/k8s.yaml',
+    'storage/k8s.yaml',
+    'muxer/k8s.yaml',
+])
+
+k8s_resource("muxer", port_forwards="8080")
+
+local_resource('update', "sed -re 's/time.Unix\(0, ([0-9]*)\)/time.Unix\(0, '\"$(($(date +%s%N)))\"'\)/' -i glitch/main.go")
```

Comment:
We repeat this a lot, can we put this in a shared Tiltfile that is include dfrom all of the other ones?

Reply:
The different versions in the `fast/` directory are snapshots at different points in time. At any given moment, that's the only instance of it, so I'd feel weird abstracting something away that for the audience effectively only appears once.